server/env.js

```diff
@@ -0,0 +1,27 @@
+import dotenv from 'dotenv';
+dotenv.config();
+
+export const {
+    DB_NAME,
+    MONGO_PASSWORD,
+    API_KEY,
+    SALT_ROUNDS
+} = process.env;
+
+// TODO: Replace with Atlas URL
```

Comment:
`docker run --rm -v mongo_data:/data/db -p 27017:27017 mongo` FTW

Reply:
yeah, deploying it on vps. So no need of atlas!